src/main/java/org/takes/rq/multipart/RqMtBase.java

```diff
@@ -140,15 +129,15 @@ public RqMtBase(final Request req) throws IOException {
         if (values == null) {
             iter = new VerboseIterable<>(
                 Collections.<Request>emptyList(),
-                new Sprintf(
+                String.format(
```

Comment:
@yegor256 I needed to reduce the coupling, and as I tested the result of both and got the same result, I believed that I could replace it. Why is it not correct? What did I miss? What does Sprintf that String.format doesn't do?